core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/AggregatedVizOpExecConfig.scala

```diff
@@ -29,7 +29,7 @@ object AggregatedVizOpExecConfig {
   ): PhysicalPlan = {
 
     val aggregateOperators =
-      AggregateOpDesc.opExecPhysicalPlan(id, List(aggFunc), groupByKeys, operatorSchemaInfo)
+      AggregateOpDesc.opExecPhysicalPlan(0, id, List(aggFunc), groupByKeys, operatorSchemaInfo)
```

Comment:
Are we using this file? If so, should we pass the correct execution id? If not, better remove this file or leave a comment.

Reply:
removed as we are not using this file any more